app/controllers/candidates_controller.rb

```diff
@@ -70,6 +73,12 @@ def get_candidate
   end
 
   def candidate_params
-    params.require(:candidate).permit(:user_id, :date_of_birth, :phone, :avatar, :cv)
+    params.require(:candidate).permit(:user_id, :date_of_birth, :phone, :avatar, :cv, user_attributes: [:id, :first_name, :last_name])
+  end
+
+  def check_authorization
+    unless current_user.id == Candidate.find(params[:id]).user_id
```

Comment:
refactor lại hàm này anh ơi :D

Reply:
Thêm cho nó cái flash[:notice] = 'Sorry, you are not a candidate this' luôn a

Reply:
okay 2 ae